modules/openxr/action_map/openxr_action.cpp

```diff
@@ -74,7 +74,7 @@ String OpenXRAction::get_name_with_set() const {
 	return action_name;
 }
 
-void OpenXRAction::set_localized_name(const String p_localized_name) {
+void OpenXRAction::set_localized_name(const String &p_localized_name) {
 	localized_name = p_localized_name;
```

Comment:
This is not necessarily an improvement. Or at least, it's not the best kind of improvement.
If you're taking in a reference (like `const String &`), you're forcing the callee to copy the value when it uses it. 
However, there's another pattern that's possible:


```suggestion
void OpenXRAction::set_localized_name(String p_localized_name) {
localized_name = std::move(p_localized_name);
```

This moves the responsibility to do the copy to the caller, who will sometimes need to do copies to pass into this function, but may sometimes be able to pass in 'fresh' objects, and no copy will be required at all.

This is, at least, the academic side of it. Godot doesn't really use move semantics much explicitly yet (though I'd kind of prefer if we did start with that). So I'm not sure if I'm suggesting you _actually_ use this paradigm instead of ref arguments here. But I wanted to weigh in that there is this _ideal_ reformatting that could be done instead.

Reply:
That's not consistent with the existing style Godot uses throughout 99% of the codebase, which is `const Type &`. I would prefer to make this code consistent with the existing dominant way Godot handles parameters.

Also, with your proposed solution, it will always make 1 copy when the function is called. If the parameter is `const Type &` then it is possible that no copies will be made at all if the value just needs to be read or transformed without making a copy, which means 0 or 1 copies, which is better than always 1 copy IMO.